examples/flow.cpp

```diff
@@ -251,15 +251,31 @@ try
     const PhaseUsage pu = Opm::phaseUsageFromDeck(deck);
     Opm::BlackoilOutputWriter outputWriter(grid, param, eclipseState, pu );
 
+    int numCells = Opm::UgGridHelpers::numCells(grid);
```

Comment:
I guess you mean cartesian -> compressed. The other one is already
there in the global index.

Concerning the this mapping, please be aware that it is not clear how
to best implement such a mapping. With few inactive cells available a
std:vector with the compressed index is optimal, with many (like in
parallel runs) it surely is not. Unfortunately this decision has to be
made during runtime depending on information outside of the grid.

-- 
Join us at the DUNE User Meeting Sept. 28-29, 2015, Heidelberg, Germany
http://users.dune-project.org/projects/dune-user-meeting-2015/wiki